.github/workflows/upload_image.yml

```diff
@@ -0,0 +1,92 @@
+name: Upload Image
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron:  '0 18 * * *'
+  release:
+    types: [published]
+
+jobs:
+  build-specific-architecture:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+        image: [chaos-daemon, chaos-mesh, chaos-dashboard, chaos-dlv]
+    outputs:
+      image_tag: ${{ steps.image_tag.outputs.image_tag }}
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Extract Image Tag
+        shell: bash
+        run: |
+          IMAGE_TAG=${GITHUB_REF##*/}
+          if [ "${IMAGE_TAG}" = "master" ] ; then
+            IMAGE_TAG=latest; 
+          fi
+
+          echo "##[set-output name=image_tag;]$(echo $IMAGE_TAG)"
```

Comment:
The format of `set-output` seems to have some problems, refs to https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-output-parameter, it should be like `echo ::set-output name=image_tag::$IMAGE_TAG`

Reply:
Fixed